website/src/pages/dashboard.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Text, useColorMode } from "@chakra-ui/react";
+import Head from "next/head";
+import { colors } from "styles/Theme/colors";
+
+import { Header } from "src/components/Header";
+import { LeaderboardWidget, SideMenu, TaskOption } from "src/components/Widgets";
+
+const Dashboard = () => {
+  const { colorMode } = useColorMode();
+  return (
+    <>
+      <Head>
+        <title>Dashboard - Open Assistant</title>
+        <meta name="description" content="Chat with Open Assistant and provide feedback." />
+      </Head>
+      <Box backgroundColor={colorMode === "light" ? colors.light.bg : colors.dark.bg} className="sm:overflow-hidden">
+        <Box className="sm:flex h-full gap-6">
+          <Box className="p-6 sm:pr-0">
+            <SideMenu />
+          </Box>
+          <Box className="flex flex-col overflow-auto p-6 sm:pl-0 gap-14">
+            <TaskOption />
+            <LeaderboardWidget />
+          </Box>
+        </Box>
+      </Box>
+    </>
+  );
+};
+
+Dashboard.getLayout = (page) => (
+  <div className="grid grid-rows-[min-content_1fr_min-content] h-full justify-items-stretch">
```

Comment:
I made a layout in `Layout.tsx` that does this, can you re-use that?

Reply:
The footer shrinks the useable space in the dashboard if we use Layout.tsx

I'd like to keep the footer off of the dashboard if possible

Reply:
Whoops, that change isn't in yet, it's in another PR.  nvm